helpers/shells/getLastSave.sh

```diff
@@ -22,4 +22,6 @@
 # Exit when any command fails
 set -e
 
-stat -c {\"user\":\"%U\",\"lastSave\":%Y\} /var/borgwarehouse/*/repos/*/integrity* | jq -s
+stat --format='{"user":"%U","lastSave":%Y}' \
+  /var/borgwarehouse/*/repos/*/integrity* |
+  jq --sort-key
```

Comment:
```suggestion
  jq --slurp
```

Reply:
Your script failed. My `-s` if for `--slurp` (it is `-S` for `--sort-key`). You can commit suggestion 👍.